templates/widget/LICENSE.mustache

```diff
@@ -0,0 +1,208 @@
+This node is under Apache-2.0 licence because it was created from the following
+fuction node code using [node-red-nodegen](https://github.com/node-red/node-red-nodegen).
```

Comment:
@HiroyasuNishiyama This description is for function node template. Please replace "function node" and URLs with node-red-dashboard template node's one.

Reply:
@kazuhitoyokoi thank you for pointing it.  fixed description in LICENSE.mustache file.